source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
Seems I get it. My assumption is following: if the developer wants to use Likely as a Commonjs module he's responsible for manual init, seems it's true in many cases. If the developer doesn't want to think about modules and makes a simple website, he just drops the script to the page and Likely works by itself (like now).